File: G2_RealArtificialSystem/ArtificialSystem/ArtificialSystemTest/src/main/java/cosn/group2/dto/DomainModel.java
package cosn.group2.dto;

public class DomainModel {

    private int id;
    public String topology;
    public int modelType;


    public DomainModel(int id, String topology, int modelType) {
        this.id = id;
        this.topology = topology;
        this.modelType = modelType;
    }

    @Override
    public String toString() {
        return "DomainModel{" +
                "id=" + id +
                ", topology='" + topology + '\'' +
                ", modelType=" + modelType +
                '}';
    }
    /*
    * “id”: 1,
    “topology”: “a”,
    “modelType”: 1
    * */
}

File: GROUP1/g01-main/DomainServiceFolder/DomainService/Models/DomainModel.cs
using System.ComponentModel.DataAnnotations;

namespace DomainService.Models
{
    public class DomainModel
    {
        [Key]
        [Required]
        public int Id { get; set; }

        [Required]
        public string Topology { get; set; }

        [Required]
        public int ModelType { get; set; }

    }
}
